.github/CODEOWNERS

```diff
@@ -16,15 +16,15 @@
 # docs/  docs@example.com
 
 #### Build stuff
-scripts/ @dzearing @ecraig12345 @xugao
+scripts/ @dzearing @ecraig12345
```

Comment:
We should also add a Prague owner for scripts stuff, but that can be discussed separately

Reply:
@Hotell would be a good choice ;-)

Reply:
Good callout, I think we add @Hotell to a lot more "Build stuff", was just trying to clean up Xu's ownership and avoid a codeowner revamp, which is needed soon anyway. 

I will add Hotell in this suggested places though.